adserver/views.py

```diff
@@ -753,28 +753,30 @@ def ignore_tracking_reason(self, request, advertisement, offer):
                 user_agent,
             )
             reason = "Ratelimited view impression"
-        if offer and offer.os_family != parsed_ua.os.family:
-            # Because this block doesn't set a reason, it will only log mismatches. Not stop them.
+        elif offer and offer.os_family != parsed_ua.os.family:
             log.log(
                 self.log_security_level,
                 "Mismatched OS between offer and impression. Publisher: [%s], Offer OS: [%s], User agent: [%s]",
                 offer.publisher,
                 offer.os_family,
                 user_agent,
             )
-        if offer and offer.browser_family != parsed_ua.browser.family:
-            # Because this block doesn't set a reason, it will only log mismatches. Not stop them.
+            reason = "Mismatched OS"
+        elif offer and offer.browser_family != parsed_ua.browser.family:
             log.log(
                 self.log_security_level,
                 "Mismatched browser between offer and impression. Publisher: [%s], Offer Browser: [%s], User agent: [%s]",
                 offer.publisher,
                 offer.browser_family,
                 user_agent,
             )
+            reason = "Mismatched browser"
+
+        # This is out of the elif block and will be run everytime
```

Comment:
Do we actually care about this happening? It seems to happen enough that I wonder if logging it is even useful -- I guess perhaps for debugging or fraud detection, but I don't know if we'll really look at it.

Reply:
By lowering the level, it won't make it into Sentry but I think this can signal that something is wrong for a publisher.